### DISCUSSION TITLE
Sync Google Ads to Postgres Normalization Error

### DISCUSSION THREAD

**Discussion by nachosan2** [airbytehq/airbyte#28474] [Connector Questions] [answered]
Hello!

I am using a self host Airbyte (0.50.7) and have connection between a Google Ads source and a Postgres destination. 

The original connection was with a Postgres 0.4.0 and a Google Ads 0.2.14 which was working smoothly, but when I updated the Google Ads to it latest version 0.7.1, although the reset didn't have any issue, the sync is failing due to a "Normalization failed during the dbt run. This may indicate a problem with the data itself." 

I am not developer so sadly I can't find any specific problem or how to solve it. I'm attaching the logs of one of the attempts which has this error message (I'm always getting the same error). The only way I thought to patch is syncronizing the data to a new schema or with a new prefix to make the difference, but I can't do that always I have this issue.

Is there anything that can be done? Or is it maybe a problem with the source or the destination connector?
Thank you very much!

(https://github.com/airbytehq/airbyte/files/12096266/70ea5c80_8f58_460b_9af6_420880dcd294_job_97_attempt_1_txt.txt)

**Comment by marcosmarxm** [comment]
Did you try to update the schema and reset the data?
```
2023-07-18 18:59:24 normalization > Completed with 1 error and 0 warnings:
2023-07-18 18:59:24 normalization > Database Error in model campaigns_stg (models/generated/airbyte_incremental/gads_0_2_14/campaigns_stg.sql)
2023-07-18 18:59:24 normalization >   syntax error at or near "."
2023-07-18 18:59:24 normalization >   LINE 6:                add column metrics.active_view_ctr double pre...
2023-07-18 18:59:24 normalization >                                            ^
```

**Reply by nachosan2** [reply]
Hi Marcos! Yes I updated the schema, reseted the data and sync again, but I got the same issue.

**Reply by marcosmarxm** [reply]
Are you able to try another destination?
Are you using a custom query for Google Ads?

**Reply by nachosan2** [reply, marked as answer]
Hey! I am not using a custom query for Google Ads. About another destination, I did create a new connection and it works fine, but it's pretty annoying if I have to do that each time I encounter this issue, so that's why I was wondering if there may be a solution.
Thanks!!

**Reply by marcosmarxm** [reply]
@nachosan2 normalization will be deprecated soon to reduce this type of problems.

**Reply by nachosan2** [reply]
Is there any estimated time to when it will available that update? Thank you!
